doc/tests/manual/protocol.md

```diff
@@ -297,6 +297,35 @@
 - Test Result:
 - Notes:
 
+## TC-018 - Edit seed
+
+- Description: Edit seed.
+- Preconditions:
+  - User is on the view seed page.
+- Test Steps:
+  1. Press Button "Edit seed"
+  2. Change an attribute of the seed.
+  3. Submit the form.
```

Comment:
Unfortunately, pressing "edit seed" is still broken on pr.permaplant.net:
`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://api/seeds/2. (Reason: CORS request did not succeed). Status code: (null).`

Reply:
If i remember correctly this is a known issue in our testing environment. The backends CORS config is setup to work on permaplant.net only. pr.permaplant.net is a different origin, causing the browser to throw CORS errors.

This is the reason why I always use chromium with disabled web security when testing locally.

@4ydan Could we set the backends server URL using an environment variable to avoid such issues in the future?

Reply:
CORS is correctly set up for pr, dev and www to work with cloud (at least for getting/putting images, there might be problems for other situations).

The problem here obviously is that the URL is broken: `https://api/seeds/2` obviously is not what we want.

> This is the reason why I always use chromium with disabled web security when testing locally.

This is unrelated. For localhost+cloud CORS is not (and probably cannot be) set up. But this has nothing to do with wrong usage of the VITE_BASE_API_URL (which is set to / on pr/dev/www). I agree this should be better documented in doc/ci.md